tracer/test/test-applications/security/Samples.Security.AspNetCore5/Views/Home/Index.cshtml

```diff
@@ -49,6 +49,11 @@
 
     <div>&nbsp;</div>
 
+    <div> <a href="/EnableASM">Enable ASM</a></div>
```

Comment:
Are these routes hooked up to any actions?

Reply:
This are remanents of past testings. Not used anymore. 
Fixed in 2f5222fe0ec432032c724578476bb9f877e1e5d1